zds/tutorialv2/tests/tests_views.py

```diff
@@ -255,6 +255,20 @@ def test_basic_tutorial_workflow(self):
         description = u'une description'
         title = u'un titre'
         random = u'un truc à la rien à voir'
+        random_with_md = u'un text contenant du **markdown** .'
+
+        response = self.client.post(
+            reverse('content:create-tutorial'),
+            {
+                'text': random_with_md,
+                'preview': '',
+            }, HTTP_X_REQUESTED_WITH='XMLHttpRequest')
+        
+
+        self.assertEqual(200, response.status_code)
+
+        result_string = ''.join(response.streaming_content)
+        self.assertTrue('<strong>markdown</strong>' in result_string)
```

Comment:
`self.assertIn('<strong>markdown</strong>', result_string, 'We need the biography to be properly formatted')`